pepper-apis/dsm-core/src/main/resources/liquibase/rgp/PEPPER-833_rgp_kit_display_name.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-833_rgp_kit_type_display_name" author="pegah">
+        <addColumn tableName="kit_type">
+            <column name="kit_display_name" type="VARCHAR(500)"></column>
```

Comment:
'display_name' is probably sufficient here, and `kit_display_name` may be confusing since it is actually the kit type display name, not the kit display name.

Reply:
Something interesting, I found out that we have an (  I am 99.9% sure ) unused column in `ddp_kit_request_settings` called `kit_type_display_name` . I think we need to investigate that more and get rid of it if it's not used. My first attempt was to get the code working with that but that would not work with sub kits